src/orange/orangeinp/ConvexRegion.hh

```diff
@@ -69,6 +69,69 @@ class Box final : public ConvexRegionInterface
     Real3 hw_;
 };
 
+//---------------------------------------------------------------------------//
+/*!
+ * A truncated cone along the Z axis centered on the origin.
```

Comment:
You should make it clear that the truncation is optional, i.e., r_lo = 0 is acceptable